californium-proxy/src/main/java/org/eclipse/californium/proxy/HttpClientFactory.java

```diff
@@ -0,0 +1,102 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Institute for Pervasive Computing, ETH Zurich and others.
```

Comment:
Greate that you use a license. But, if you'r not working for the ETH, I woudl prefer to have either you name, or if you did it at work, the name of your company.